src/main/java/frc/robot/subsystems/drivetrain/tests/AngleMotorsSpeedTest.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems.drivetrain.tests;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+public class AngleMotorsSpeedTest extends CommandBase {
+    private final SwerveDrive swerveDrive;
+    private final Timer timer = new Timer();
+    private final double times[] = new double[4];
+
+    public AngleMotorsSpeedTest(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        swerveDrive.setStates(new SwerveModuleState[]{
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90))
+        });
```

Comment:
Consider the following:
```suggestion
        SwerveModuleState[] states = new SwerveModuleState[4];
        Arrays.fill(states, new SwerveModuleState(0, Rotation2d.fromDegrees(90)));
```
Also, because the states don't change, I would create this array in the constructor or in `initialize()` and only reference it in `execute()`.